Paper.tex

```diff
@@ -830,7 +830,7 @@ \subsection{Execution}
 a \cup T_{\mathrm{t}} & \text{if} \quad T_{\mathrm{t}} \neq \varnothing \\
 a & \text{otherwise}
 \end{cases} \\
-a & \equiv A^0_{\mathbf{a}} \cup \{S(T)\} \cup_{E \in T_{\mathbf{A}}} \{ \hyperlink{access_list_entry}{E}_{\mathrm{a}} \}
+a & \equiv A^0_{\mathbf{a}} \cup \{S(T)\} \cup \{H_c\}\cup_{E \in T_{\mathbf{A}}} \{ \hyperlink{access_list_entry}{E}_{\mathrm{a}} \}
```

Comment:
This part LGTM.

Small observations:
1) Are the curly brackets necessary? (i.e. can `\{H_c\}` just be  `H_c`). I guess because the union operation is defined on sets, if that is the criteria then formula `(74)` on `a ∪ T_t` is missing the brackets.
2) My understanding is that this EIP is retro-compatible, still should it specify in any comment/footnote that it was introduced with `EIP-3651`?
3) (_Out of scope of the PR_) While reading the Block definition section 4.3, `H_c` could also state the following: "(also known as the coinbase address)"

Reply:
Re: EIP-3651
1. Looking at other formulas of the YP, you are right, I just removed `{ }` for clarity
2. I don't think so, EIPs are rarely cited except for major changes such as EIP1559.
3. Totally agree with you, I didn't added it though, cause it's out of scope and I don't want to add more noise, this PR will already be hard to merge, might be worth it to add it as a tiny separate PR.

Reply:
Changes in [689eb33](https://github.com/ethereum/yellowpaper/pull/906/commits/689eb3353be76717d1378640b06551d73826d6a4)